lighthouse-extension/app/src/popup.js

```diff
@@ -53,7 +53,7 @@ document.addEventListener('DOMContentLoaded', _ => {
     qsBody += '**Stack Trace**:\n ```' + err.stack + '```';
 
     const base = 'https://github.com/googlechrome/lighthouse/issues/new?';
-    const title = encodeURI('title=Lighthouse Extension Error');
+    const title = encodeURI('title=Extension Error: ' + err.message);
     const body = '&body=' + encodeURI(qsBody);
```

Comment:
I feel like this would benefit from some upper bound truncation in case the `err.message` gets a little long.